appscale/tools/appscale.py

```diff
@@ -85,7 +92,7 @@ class AppScale():
   get <regex>                       Gets all AppController properties matching
                                     the provided regex: for developers only.
   help                              Displays this message.
-  init <cloud|cluster>              Writes a new configuration file for
+  init [cloud|cluster]              Writes a new configuration file for
```

Comment:
Do we need the cloud|cluster at all? WHat about we remove it from the description, but we still tolerate for backward compatibility?